README.md

```diff
@@ -0,0 +1,222 @@
+<h1 align="center">lumino-tests</h1>
+
+<h2 style="display: inline-block">Table of Contents</h2>
+<ol>
+  <li><a href="#about">About</a>
+    <ul>
+      <li><a href="#how-it-works">How it Works</a></li>
+    </ul>
+  </li>
+  <li>
+    <a href="#getting-started">Getting Started</a>
+    <ul>
+      <li><a href="#prerequisites">Prerequisites</a>
+        <ul>
+          <li><a href="#nodejs--npm">Node.js & npm</a></li>
+          <li><a href="#typescript">TypeScript</a></li>
+          <li><a href="#docker">Docker</a></li>
+          <li><a href="#lumino-sdk">Lumino SDK</a></li>
+        </ul>
+      </li>
+      <li><a href="#installation">Installation</a>
+        <ul>
+          <li><a href="#repo">Repo</a></li>
+          <li><a href="#docker-images">Docker Images</a></li>
+          <li><a href="#project">Project</a></li>
+        </ul>
+      </li>
+    </ul>
+  </li>
+  <li><a href="#usage">Usage</a>
+    <ul>
+      <li><a href="#test-suite">Test Suite</a>
+        <ul>
+          <li><a href="#individual-tests">Individual Tests</a></li>
+        </ul>
+      </li>
+      <li><a href="#new-tests">New Tests</a>
+      <li><a href="#rebuild-docker-images">Rebuild Docker Images</a>
+    </ul>
+  </li>
+  <li><a href="#external-links">External Links</a></li>
+</ol>
+
+## About
+
+The purpose of the `lumino-tests` project is to provide tools which can run arbitrarily complex automated tests. These aim to serve as complements to the RIF Lumino unit tests, for robustness purposes.
+
+### How it Works
+
+The project is written chiefly in [TypeScript](https://www.typescriptlang.org/) and uses [Jest](https://jestjs.io/) as its testing framework.
+
+Under the hood it makes use of both [Docker](https://www.docker.com/), as well as the [Lumino SDK](https://github.com/rsksmart/lumino-sdk), in order to boot up testing scenarios and carry out actions in them.
+
+## Getting Started
+
+Please note that all instructions are written for Ubuntu.
+
+### Prerequisites
+
+The following prerequisites and dependencies must be installed manually beforehand.
+
+#### Node.js & `npm`
+
+These should be included with your Ubuntu distribution. `node v14.x` and `npm 7.x` should be more than enough. You can check these by executing:
+
+```bash
+node -v
+npm -v
+```
+
+The use of [`nvm`](https://github.com/nvm-sh/nvm) is **recommended**.
+
+If you have any doubts please check the official documentation here:
+
+- [Node.js](https://nodejs.org/en/)
+- [npm | get npm](https://www.npmjs.com/get-npm)
+- [nvm-sh/nvm: Node Version Manager](https://github.com/nvm-sh/nvm)
+
+#### TypeScript
+
+Install TypeScript by running the following terminal command:
+
+```
+npm install typescript
```

Comment:
I think this is not needed

Reply:
it shouldn't no, it should be on the package.json file

Reply:
you're right. i remember doing this, but it was actually for the typescript course, not this project.

my bad! fixed